lib/doorkeeper/models/concerns/revocable.rb

```diff
@@ -8,6 +8,29 @@ def revoke(clock = Time)
       def revoked?
         !!(revoked_at && revoked_at <= Time.now)
       end
+
+      def revoke_in(time)
+        update_attribute :revoked_at, Time.now + time
```

Comment:
> Should we ever be using Time.now as opposed to Time.now.utc

I think it's best to keep consistent. Which makes me want to merge your other PR which makes everything UTC.